android/src/main/java/com/getcapacitor/community/firebaseanalytics/FirebaseAnalytics.java

```diff
@@ -147,11 +147,10 @@ public void setScreenName(final PluginCall call) {
 
             @Override
             public void run() {
-              mFirebaseAnalytics.setCurrentScreen(
-                bridge.getActivity(),
-                screenName,
-                nameOverride
-              );
+              Bundle bundle = new Bundle();
+              bundle.putString(FirebaseAnalytics.Param.SCREEN_NAME, screenName);
```

Comment:
Don't you think it should be `com.google.firebase.analytics.FirebaseAnalytics.Param.SCREEN_NAME` here ? As `FirebaseAnalytics` is the current class ?

Reply:
This has been fixed with release 0.3.0